voltdb/manifest.json

```
@@ -1,20 +1,20 @@
 {
-  "display_name": "VoltDB",
+  "display_name": "Volt Active Data",
```

```suggestion
  "display_name": "VoltDB",
```
This cannot be changed (think of it like an ID) but it is not a problem since the value from `public_title` is the one that will be shown (It it's confusing, I know)